Userland/Libraries/LibCore/System.cpp

```diff
@@ -66,6 +66,71 @@ ErrorOr<void> beep()
     return {};
 }
 
+String Promises::to_string() const
+{
+    Vector<String> promises_builder;
+
+    // FIXME: An enumeration macro seems overkill?
```

Comment:
There is already an enumeration macro for this in the kernel, if we are going to go this route then you might as well move it to a shared header, so you can take advantage of it from userland as well. See: `ENUMERATE_PLEDGE_PROMISES`

Reply:
Yeah I saw that but didn't know whether I should/could share that macro.